src/main/java/pl/orki/hackathon/webapp/user/control/UserService.java

```diff
@@ -31,4 +51,20 @@ public User createUser(User user) {
         user.setPassword(bCryptPasswordEncoder.encode(user.getPassword()));
         return userRepository.save(user);
     }
+
+    public UserResponseDTO createUserBand(UserDTO userDTO, BandDTO bandDTO) {
```

Comment:
Add `@Transactional` annotation

Reply:
I was thinking about returning entity instead of dto form service, and convert it withing `UserMutation`, but it's up to you ;)